lib/i18n-transform.js

```diff
@@ -28,41 +28,51 @@ var _getLanguageFromI18n = function (i18nObj) {
         return null;
     },
 
-    _isWildcard = function (language) {
+    _isWildcardMatch = function (language) {
         return language.code === "*";
     },
 
-    _isLanguageCodeMatch = function (i18nLanguage, language) {
-        return language.code.toUpperCase() === i18nLanguage.code.toUpperCase();
+    _isLanguageCodeMatch = function (languageA, languageB) {
+        return languageA.code.toUpperCase() === languageB.code.toUpperCase();
```

Comment:
Quick question, we have a null check for .region but none for .code, any reason for that?